app/views/providers/legal_aid_applications/_application_list.html.erb

```diff
@@ -1,86 +1,91 @@
+        table.with_body do |body|
+          @legal_aid_applications.each do |application|
+            body.with_row do |row|
+              row.with_cell { link_to_application(application.applicant_full_name, application) }
+              row.with_cell(text: l(application.created_at.to_date, format: :short_date))
+              row.with_cell { application.application_ref }
+              row.with_cell do
+                if application.special_children_act_proceedings? && application.used_delegated_functions?
+                  concat t(".substantive")
                   concat tag.span(class: "govuk-body-s govuk-!-display-block") {
-                    t(".substantive_due", date: l(application.substantive_application_deadline_on, format: :short_date))
+                    t(".backdated_to", date: l(application.used_delegated_functions_on, format: :short_date))
                   }
+                elsif application.used_delegated_functions?
+                  concat t(".emergency")
+                  if application.substantive_application_deadline_on? && application.summary_state == :in_progress
+                    concat tag.span(class: "govuk-body-s govuk-!-display-block") {
+                      t(".substantive_due", date: l(application.substantive_application_deadline_on, format: :short_date))
+                    }
+                  end
+                else
+                  t(".substantive")
                 end
-              else
-                t(".substantive")
               end
-            end
-            if @scope == :in_progress
-              row.with_cell do
-                unless application.summary_state == :submitted
-                  concat govuk_button_link_to(
-                    t(".delete"),
-                    providers_legal_aid_application_delete_path(application),
-                    class: "govuk-!-margin-1",
-                    secondary: true,
-                    suffix: t(".delete_suffix", reference: application.application_ref, applicant: application.applicant_full_name),
-                  )
+              if @scope == :in_progress
+                row.with_cell do
+                  unless application.summary_state == :submitted
+                    govuk_link_to(
+                      t(".delete"),
+                      providers_legal_aid_application_delete_path(application),
+                      secondary: true,
+                      suffix: t(".delete_suffix", reference: application.application_ref, applicant: application.applicant_full_name),
+                    )
+                  end
                 end
               end
             end
           end
         end
-      end
-    end %>
+      end %>
+<% else %>
+  <p class="govuk-body"><%= t(".no_applications_html") %></p>
+<% end %>
 
-<% if @scope == :in_progress %>
+<% if @scope == :in_progress && LegalAidApplication.any?(&:expired?) %>
```

Comment:
This looks like if _any_ applications in the service are expired rather than just with the current firm

Reply:
Ive fixed this now